crates/holochain_types/src/app/app_manifest/app_manifest_v1.rs

```diff
@@ -82,11 +82,7 @@ impl AppRoleManifest {
 pub struct AppRoleDnaManifest {
     /// Where to find this Dna. To specify a DNA included in a hApp Bundle,
     /// use a local relative path that corresponds with the bundle structure.
-    ///
-    /// Note that since this is flattened,
-    /// there is no actual "location" key in the manifest.
-    #[serde(flatten)]
-    pub location: Option<mr_bundle::Location>,
+    pub file: Option<String>,
```

Comment:
Would be good to note in the changelog as a breaking change.

Reply:
Yes, this should have been mentioned in the `holochain_types` changelog